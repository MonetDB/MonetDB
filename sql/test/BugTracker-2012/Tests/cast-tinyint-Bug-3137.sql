select cast('10' as tinyint) as not_ok_tinyint, cast('10' as int) as ok_int, cast('10' as smallint) as ok_smallint;
