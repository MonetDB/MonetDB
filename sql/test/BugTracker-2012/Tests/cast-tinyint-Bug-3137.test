query III rowsort
select cast('10' as tinyint) as not_ok_tinyint, cast('10' as int) as ok_int, cast('10' as smallint) as ok_smallint
----
10
10
10

