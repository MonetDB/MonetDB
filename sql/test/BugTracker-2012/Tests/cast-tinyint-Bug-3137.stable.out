stdout of test 'cast-tinyint-Bug-3137` in directory 'sql/test/BugTracker-2012` itself:


# 16:45:44 >  
# 16:45:44 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/ufs/sjoerd/Monet-stable/var/MonetDB" "--set" "mapi_open=true" "--set" "mapi_port=34229" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_test_BugTracker-2012" "--set" "mal_listing=0"
# 16:45:44 >  

# MonetDB 5 server v11.11.8 (hg id: 087fd3ac9177)
# This is an unreleased version
# Serving database 'mTests_test_BugTracker-2012', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.629 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://madrid.ins.cwi.nl:34229/
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded

# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_history.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_compress.sql
# loading sql script: 18_dictionary.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 39_analytics.sql
# loading sql script: 40_geom.sql
# loading sql script: 80_udf.sql
# loading sql script: 99_system.sql

# 16:45:44 >  
# 16:45:44 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=madrid" "--port=34229"
# 16:45:44 >  

#select cast('10' as tinyint) as not_ok_tinyint, cast('10' as int) as ok_int, cast('10' as smallint) as ok_smallint;
% .,	.,	. # table_name
% not_ok_tinyint,	ok_int,	ok_smallint # name
% tinyint,	int,	smallint # type
% 2,	2,	2 # length
[ 10,	10,	10	]

# 16:45:44 >  
# 16:45:44 >  "Done."
# 16:45:44 >  

